File: app/src/main/java/org/fossasia/susi/ai/adapters/viewholders/TypingDotsHolder.java
package org.fossasia.susi.ai.adapters.viewholders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.LinearLayout;

import org.fossasia.susi.ai.R;

import butterknife.BindView;
import butterknife.ButterKnife;
import pl.tajchert.sample.DotsTextView;

/**
 * <h1>Typing dots view holder</h1>
 *
 * Created by mejariamol on 10/21/2016.
 */
public class TypingDotsHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.dots)
    public DotsTextView dotsTextView;
    @BindView(R.id.background_layout)
    public LinearLayout backgroundLayout;

    /**
     * Instantiates a new Typing dots holder.
     *
     * @param itemView the item view
     */
    public TypingDotsHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/TypingDotsHolder.kt
package org.fossasia.susi.ai.chat.adapters.viewholders

import android.view.View
import android.widget.LinearLayout
import androidx.recyclerview.widget.RecyclerView
import kotterknife.bindView
import org.fossasia.susi.ai.R
import pl.tajchert.waitingdots.DotsTextView

class TypingDotsHolder(itemView: View) : androidx.recyclerview.widget.RecyclerView.ViewHolder(itemView) {

    val dotsTextView: DotsTextView by bindView(R.id.dots)
    val backgroundLayout: LinearLayout by bindView(R.id.background_layout)
}
